spec/ConfigChanges/ConfigFile.spec.js

```diff
@@ -19,73 +19,83 @@ var rewire = require('rewire');
+                spyOn(fs, 'readFileSync').and.returnValue('bplist');
+                expect(configFile.isBinaryPlist('someFile')).toBe(true);
+            });
+        });
 
-    it('getIOSProjectname should throw error', function () {
-        expect(function () { configFile.getIOSProjectname('some/project/name'); }).toThrow();
-    });
+        describe('getIOSProjectname', () => {
+            it('should throw error', function () {
+                expect(function () { configFile.getIOSProjectname('some/project/name'); }).toThrow();
+            });
+        });
 
-    it('resolveConfigFilePath should return file path', function () {
-        var filePath = path.join('project_dir', 'file');
-        expect(configFile.resolveConfigFilePath('project_dir', 'platform', 'file')).toBe(filePath);
-    });
+        describe('resolveConfigFilePath', () => {
+            const projectDir = path.join('project_dir', 'app', 'src', 'main');
 
-    it('resolveConfigFilePath should return file path', function () {
-        var androidManifestPath = path.join(projectDir, 'AndroidManifest.xml');
-        expect(configFile.resolveConfigFilePath('project_dir', 'android', 'AndroidManifest.xml')).toBe(androidManifestPath);
-    });
+            it('should return file path', function () {
+                var filePath = path.join('project_dir', 'file');
+                expect(configFile.resolveConfigFilePath('project_dir', 'platform', 'file')).toBe(filePath);
+            });
 
-    it('resolveConfigFilePath should return file path', function () {
-        var configPath = path.join(projectDir, 'res', 'xml', 'config.xml');
-        expect(configFile.resolveConfigFilePath('project_dir', 'android', 'config.xml')).toBe(configPath);
-    });
+            it('should return file path', function () {
+                var androidManifestPath = path.join(projectDir, 'AndroidManifest.xml');
+                expect(configFile.resolveConfigFilePath('project_dir', 'android', 'AndroidManifest.xml')).toBe(androidManifestPath);
+            });
 
-    it('resolveConfigFilePath should return file path', function () {
-        var stringsPath = path.join(projectDir, 'res', 'values', 'strings.xml');
-        expect(configFile.resolveConfigFilePath('project_dir', 'android', 'strings.xml')).toBe(stringsPath);
-    });
+            it('should return file path', function () {
+                var configPath = path.join(projectDir, 'res', 'xml', 'config.xml');
+                expect(configFile.resolveConfigFilePath('project_dir', 'android', 'config.xml')).toBe(configPath);
+            });
 
-    it('resolveConfigFilePath should return file path', function () {
-        spyOn(configFile, 'getIOSProjectname').and.returnValue('iospath');
-        var configPath = path.join('project_dir', 'iospath', 'config.xml');
-        expect(configFile.resolveConfigFilePath('project_dir', 'ios', 'config.xml')).toBe(configPath);
-    });
+            it('should return file path', function () {
```

Comment:
We already have a test `should return file path ` in this structure.
Could it be updated to provide some uniqueness? e.g. `should return strings.xml  file path`

Reply:
```suggestion
            it('should return strings.xml file path', function () {
```